giskard/testing/tests/data_quality.py

```diff
@@ -0,0 +1,64 @@
+"""
+Module for data quality tests.
+"""
+from giskard.ml_worker.testing.test_result import TestResult
+from giskard.ml_worker.testing.registry.decorators import test
+from giskard.datasets.base import Dataset
+
+@test(name="Data Uniqueness Test")
+def uniqueness_test(dataset: Dataset, column: str):
+    """
+    Test for checking the uniqueness of data in a column.
+
+    Args:
+        dataset (Dataset): The dataset to test.
+        column (str): The column to check for uniqueness.
+
+    Returns:
+        TestResult: The result of the test.
+    """
+    column_data = dataset.df[column]
+    uniqueness_ratio = len(column_data.unique()) / len(column_data)
+    return TestResult(passed=uniqueness_ratio == 1)
```

Comment:
Nit picking here but in Giskard you can put a metric, this way when a test fail we can easily know what was our actual score

```suggestion
    return TestResult(passed=uniqueness_ratio == 1, metric=uniqueness_ratio)
```